### PR TITLE
Summary rows no row groups

### LINKED ISSUE #1292: tidyselect error in `gt::summary_rows()` without groups (in v0.9.0)
## Prework

* [x] Read and agree to the [code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html) and [contributing guidelines](https://github.com/rstudio/gt/blob/master/.github/CONTRIBUTING.md).
* [x] If there is [already a relevant issue](https://github.com/rstudio/gt/issues), whether open or closed, comment on the existing thread instead of posting a new issue.

## Description

See the error message. 

## Reproducible example

``` r
mtcars |>
        dplyr::select(gear) |>
        gt::gt(rownames_to_stub = TRUE) |>
        gt::summary_rows(fns = "sum")
#> Error:
#> ! `everything()` must be used within a *selecting* function.
#> ℹ See <https://tidyselect.r-lib.org/reference/faq-selection-context.html> for
#>   details.
#> Backtrace:
#>     ▆
#>  1. └─gt::summary_rows(...)
#>  2.   └─tidyselect::everything()
#>  3.     ├─vars %||% peek_vars(fn = "everything")
#>  4.     └─tidyselect::peek_vars(fn = "everything")
#>  5.       └─cli::cli_abort(...)
#>  6.         └─rlang::abort(...)
```

<sup>Created on 2023-04-04 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

## Expected result

I'd expect the above expression to return the same as `gt::grand_summary_rows(fns = "sum")`

<details><summary>Reprex:</summary>

``` r
mtcars |>
        dplyr::select(gear) |>
        gt::gt(rownames_to_stub = TRUE) |>
        gt::grand_summary_rows(fns = "sum")
```

<div id="toywfipzsh" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    &#10;    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id=""></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" scope="col" id="gear">gear</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><th id="stub_1_1" scope="row" class="gt_row gt_left gt_stub">Mazda RX4</th>
<td headers="stub_1_1 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_2" scope="row" class="gt_row gt_left gt_stub">Mazda RX4 Wag</th>
<td headers="stub_1_2 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_3" scope="row" class="gt_row gt_left gt_stub">Datsun 710</th>
<td headers="stub_1_3 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_4" scope="row" class="gt_row gt_left gt_stub">Hornet 4 Drive</th>
<td headers="stub_1_4 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_5" scope="row" class="gt_row gt_left gt_stub">Hornet Sportabout</th>
<td headers="stub_1_5 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_6" scope="row" class="gt_row gt_left gt_stub">Valiant</th>
<td headers="stub_1_6 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_7" scope="row" class="gt_row gt_left gt_stub">Duster 360</th>
<td headers="stub_1_7 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_8" scope="row" class="gt_row gt_left gt_stub">Merc 240D</th>
<td headers="stub_1_8 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_9" scope="row" class="gt_row gt_left gt_stub">Merc 230</th>
<td headers="stub_1_9 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_10" scope="row" class="gt_row gt_left gt_stub">Merc 280</th>
<td headers="stub_1_10 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_11" scope="row" class="gt_row gt_left gt_stub">Merc 280C</th>
<td headers="stub_1_11 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_12" scope="row" class="gt_row gt_left gt_stub">Merc 450SE</th>
<td headers="stub_1_12 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_13" scope="row" class="gt_row gt_left gt_stub">Merc 450SL</th>
<td headers="stub_1_13 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_14" scope="row" class="gt_row gt_left gt_stub">Merc 450SLC</th>
<td headers="stub_1_14 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_15" scope="row" class="gt_row gt_left gt_stub">Cadillac Fleetwood</th>
<td headers="stub_1_15 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_16" scope="row" class="gt_row gt_left gt_stub">Lincoln Continental</th>
<td headers="stub_1_16 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_17" scope="row" class="gt_row gt_left gt_stub">Chrysler Imperial</th>
<td headers="stub_1_17 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_18" scope="row" class="gt_row gt_left gt_stub">Fiat 128</th>
<td headers="stub_1_18 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_19" scope="row" class="gt_row gt_left gt_stub">Honda Civic</th>
<td headers="stub_1_19 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_20" scope="row" class="gt_row gt_left gt_stub">Toyota Corolla</th>
<td headers="stub_1_20 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_21" scope="row" class="gt_row gt_left gt_stub">Toyota Corona</th>
<td headers="stub_1_21 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_22" scope="row" class="gt_row gt_left gt_stub">Dodge Challenger</th>
<td headers="stub_1_22 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_23" scope="row" class="gt_row gt_left gt_stub">AMC Javelin</th>
<td headers="stub_1_23 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_24" scope="row" class="gt_row gt_left gt_stub">Camaro Z28</th>
<td headers="stub_1_24 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_25" scope="row" class="gt_row gt_left gt_stub">Pontiac Firebird</th>
<td headers="stub_1_25 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_26" scope="row" class="gt_row gt_left gt_stub">Fiat X1-9</th>
<td headers="stub_1_26 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_27" scope="row" class="gt_row gt_left gt_stub">Porsche 914-2</th>
<td headers="stub_1_27 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_28" scope="row" class="gt_row gt_left gt_stub">Lotus Europa</th>
<td headers="stub_1_28 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_29" scope="row" class="gt_row gt_left gt_stub">Ford Pantera L</th>
<td headers="stub_1_29 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_30" scope="row" class="gt_row gt_left gt_stub">Ferrari Dino</th>
<td headers="stub_1_30 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_31" scope="row" class="gt_row gt_left gt_stub">Maserati Bora</th>
<td headers="stub_1_31 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_32" scope="row" class="gt_row gt_left gt_stub">Volvo 142E</th>
<td headers="stub_1_32 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="grand_summary_stub_1" scope="row" class="gt_row gt_left gt_stub gt_grand_summary_row gt_first_grand_summary_row gt_last_summary_row">sum</th>
<td headers="grand_summary_stub_1 gear" class="gt_row gt_right gt_grand_summary_row gt_first_grand_summary_row gt_last_summary_row">118</td></tr>
  </tbody>
  &#10;  
</table>
</div>

<sup>Created on 2023-04-04 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

</details>

## Workaround

One can set `groups` to the undocumented sentinel `":GRAND_SUMMARY:"`:

<details><summary>Reprex:</summary>

``` r
mtcars |>
        dplyr::select(gear) |>
        gt::gt(rownames_to_stub = TRUE) |>
        gt::summary_rows(fns = "sum",
                         groups = ":GRAND_SUMMARY:")
```

<div id="csnnjlxvuf" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    &#10;    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id=""></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" scope="col" id="gear">gear</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><th id="stub_1_1" scope="row" class="gt_row gt_left gt_stub">Mazda RX4</th>
<td headers="stub_1_1 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_2" scope="row" class="gt_row gt_left gt_stub">Mazda RX4 Wag</th>
<td headers="stub_1_2 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_3" scope="row" class="gt_row gt_left gt_stub">Datsun 710</th>
<td headers="stub_1_3 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_4" scope="row" class="gt_row gt_left gt_stub">Hornet 4 Drive</th>
<td headers="stub_1_4 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_5" scope="row" class="gt_row gt_left gt_stub">Hornet Sportabout</th>
<td headers="stub_1_5 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_6" scope="row" class="gt_row gt_left gt_stub">Valiant</th>
<td headers="stub_1_6 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_7" scope="row" class="gt_row gt_left gt_stub">Duster 360</th>
<td headers="stub_1_7 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_8" scope="row" class="gt_row gt_left gt_stub">Merc 240D</th>
<td headers="stub_1_8 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_9" scope="row" class="gt_row gt_left gt_stub">Merc 230</th>
<td headers="stub_1_9 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_10" scope="row" class="gt_row gt_left gt_stub">Merc 280</th>
<td headers="stub_1_10 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_11" scope="row" class="gt_row gt_left gt_stub">Merc 280C</th>
<td headers="stub_1_11 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_12" scope="row" class="gt_row gt_left gt_stub">Merc 450SE</th>
<td headers="stub_1_12 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_13" scope="row" class="gt_row gt_left gt_stub">Merc 450SL</th>
<td headers="stub_1_13 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_14" scope="row" class="gt_row gt_left gt_stub">Merc 450SLC</th>
<td headers="stub_1_14 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_15" scope="row" class="gt_row gt_left gt_stub">Cadillac Fleetwood</th>
<td headers="stub_1_15 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_16" scope="row" class="gt_row gt_left gt_stub">Lincoln Continental</th>
<td headers="stub_1_16 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_17" scope="row" class="gt_row gt_left gt_stub">Chrysler Imperial</th>
<td headers="stub_1_17 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_18" scope="row" class="gt_row gt_left gt_stub">Fiat 128</th>
<td headers="stub_1_18 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_19" scope="row" class="gt_row gt_left gt_stub">Honda Civic</th>
<td headers="stub_1_19 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_20" scope="row" class="gt_row gt_left gt_stub">Toyota Corolla</th>
<td headers="stub_1_20 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_21" scope="row" class="gt_row gt_left gt_stub">Toyota Corona</th>
<td headers="stub_1_21 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_22" scope="row" class="gt_row gt_left gt_stub">Dodge Challenger</th>
<td headers="stub_1_22 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_23" scope="row" class="gt_row gt_left gt_stub">AMC Javelin</th>
<td headers="stub_1_23 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_24" scope="row" class="gt_row gt_left gt_stub">Camaro Z28</th>
<td headers="stub_1_24 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_25" scope="row" class="gt_row gt_left gt_stub">Pontiac Firebird</th>
<td headers="stub_1_25 gear" class="gt_row gt_right">3</td></tr>
    <tr><th id="stub_1_26" scope="row" class="gt_row gt_left gt_stub">Fiat X1-9</th>
<td headers="stub_1_26 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="stub_1_27" scope="row" class="gt_row gt_left gt_stub">Porsche 914-2</th>
<td headers="stub_1_27 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_28" scope="row" class="gt_row gt_left gt_stub">Lotus Europa</th>
<td headers="stub_1_28 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_29" scope="row" class="gt_row gt_left gt_stub">Ford Pantera L</th>
<td headers="stub_1_29 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_30" scope="row" class="gt_row gt_left gt_stub">Ferrari Dino</th>
<td headers="stub_1_30 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_31" scope="row" class="gt_row gt_left gt_stub">Maserati Bora</th>
<td headers="stub_1_31 gear" class="gt_row gt_right">5</td></tr>
    <tr><th id="stub_1_32" scope="row" class="gt_row gt_left gt_stub">Volvo 142E</th>
<td headers="stub_1_32 gear" class="gt_row gt_right">4</td></tr>
    <tr><th id="grand_summary_stub_1" scope="row" class="gt_row gt_left gt_stub gt_grand_summary_row gt_first_grand_summary_row gt_last_summary_row">sum</th>
<td headers="grand_summary_stub_1 gear" class="gt_row gt_right gt_grand_summary_row gt_first_grand_summary_row gt_last_summary_row">118</td></tr>
  </tbody>
  &#10;  
</table>
</div>

<sup>Created on 2023-04-04 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

</details>

## Session info

<details style="margin-bottom:10px;">
<summary>
Session info
</summary>

``` r
sessioninfo::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value
#>  version  R version 4.2.3 (2023-03-15)
#>  os       Ubuntu 22.04.2 LTS
#>  system   x86_64, linux-gnu
#>  ui       X11
#>  language en_US.utf8
#>  collate  en_US.utf8
#>  ctype    en_US.utf8
#>  tz       Europe/Zurich
#>  date     2023-04-04
#>  pandoc   3.1.2 @ /usr/bin/ (via rmarkdown)
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version date (UTC) lib source
#>  cli           3.6.1   2023-03-23 [1] RSPM (R 4.2.0)
#>  digest        0.6.31  2022-12-11 [1] RSPM (R 4.2.0)
#>  dplyr         1.1.1   2023-03-22 [1] CRAN (R 4.2.3)
#>  evaluate      0.20    2023-01-17 [1] RSPM (R 4.2.0)
#>  fansi         1.0.4   2023-01-22 [1] RSPM (R 4.2.0)
#>  fastmap       1.1.1   2023-02-24 [1] RSPM (R 4.2.0)
#>  fs            1.6.1   2023-02-06 [1] RSPM (R 4.2.0)
#>  generics      0.1.3   2022-07-05 [1] RSPM (R 4.2.0)
#>  glue          1.6.2   2022-02-24 [1] RSPM (R 4.2.0)
#>  gt            0.9.0   2023-03-31 [1] RSPM (R 4.2.0)
#>  htmltools     0.5.5   2023-03-23 [1] RSPM (R 4.2.0)
#>  knitr         1.42    2023-01-25 [1] RSPM (R 4.2.0)
#>  lifecycle     1.0.3   2022-10-07 [1] RSPM (R 4.2.0)
#>  magrittr      2.0.3   2022-03-30 [1] RSPM (R 4.2.0)
#>  pillar        1.9.0   2023-03-22 [1] RSPM (R 4.2.0)
#>  pkgconfig     2.0.3   2019-09-22 [1] RSPM (R 4.2.0)
#>  purrr         1.0.1   2023-01-10 [1] RSPM (R 4.2.0)
#>  R.cache       0.16.0  2022-07-21 [1] RSPM (R 4.2.0)
#>  R.methodsS3   1.8.2   2022-06-13 [1] RSPM (R 4.2.0)
#>  R.oo          1.25.0  2022-06-12 [1] RSPM (R 4.2.0)
#>  R.utils       2.12.2  2022-11-11 [1] RSPM (R 4.2.0)
#>  R6            2.5.1   2021-08-19 [1] RSPM (R 4.2.0)
#>  reprex        2.0.2   2022-08-17 [1] RSPM (R 4.2.0)
#>  rlang         1.1.0   2023-03-14 [1] RSPM (R 4.2.0)
#>  rmarkdown     2.21    2023-03-26 [1] RSPM (R 4.2.0)
#>  rstudioapi    0.14    2022-08-22 [1] RSPM (R 4.2.0)
#>  sessioninfo   1.2.2   2021-12-06 [1] RSPM (R 4.2.0)
#>  styler        1.9.1   2023-03-04 [1] RSPM (R 4.2.0)
#>  tibble        3.2.1   2023-03-20 [1] RSPM (R 4.2.0)
#>  tidyselect    1.2.0   2022-10-10 [1] RSPM (R 4.2.0)
#>  utf8          1.2.3   2023-01-31 [1] RSPM (R 4.2.0)
#>  vctrs         0.6.1   2023-03-22 [1] RSPM (R 4.2.0)
#>  withr         2.5.0   2022-03-03 [1] RSPM (R 4.2.0)
#>  xfun          0.38    2023-03-24 [1] RSPM (R 4.2.0)
#>  xml2          1.3.3   2021-11-30 [1] RSPM (R 4.2.0)
#>  yaml          2.3.7   2023-01-23 [1] RSPM (R 4.2.0)
#> 
#>  [1] /home/salim/.software/managed/r
#>  [2] /usr/local/lib/R/site-library
#>  [3] /usr/lib/R/site-library
#>  [4] /usr/lib/R/library
#> 
#> ──────────────────────────────────────────────────────────────────────────────
```

</details>

### PR DESCRIPTION
**Pull request by olivroy**
Fix #1292 https://github.com/rstudio/gt/discussions/1420

It is now recommended to use `grand_summary_rows()`. The error message does a better job to point to the correct solution.

```r
# before
      mtcars %>% dplyr::select(gear) %>% gt::gt(rownames_to_stub = TRUE) %>% gt::summary_rows(
        fns = "sum")
#>      Error:
#>      ! `everything()` must be used within a *selecting* function.
#>     i See <https://tidyselect.r-lib.org/reference/faq-selection-context.html> for details.


# this PR
#>      Error:
#>      ! Since gt v0.9.0, the `groups = everything()` option has been deprecated in `summary_rows()` if no row #> groups are present.
#>       * Use `grand_summary_rows()` instead or add row groups.

### CODE REVIEW

File: R/resolver.R [hunk 1]
```diff
@@ -556,7 +556,22 @@ resolve_groups <- function(expr, vector) {
...
 
+  # Handle groups = FALSE supplied to not do any summary rows.
+  if (isFALSE(resolved)) {
+    resolved <- NULL
+  }
+
   if (length(resolved) < 1) {
+    # 1292
+    input <- tryCatch(rlang::as_label(quo), error = NULL)
+    if (identical(input, "everything()")) {
```
Comment (olivroy) [inline R/resolver.R:567]: The only way I found to figure out if the call was from `grand_summary_rows()` or not.

File: R/resolver.R [hunk 2]
```diff
@@ -556,7 +556,22 @@ resolve_groups <- function(expr, vector) {
...
+      cli::cli_warn(c(
+        "Since gt v0.9.0, the `groups = everything()` option has been deprecated in {.fn summary_rows}.",
+        "*" = "If this was intended for generation of grand summary rows, instead
+  use the `grand_summary_rows()` function."
+      ))
+      return(":GRAND_SUMMARY:")
```
Comment (olivroy) [inline R/resolver.R:573]:
```suggestion
      cli::cli_abort(c(
        "Since gt v0.9.0, the `groups = everything()` option has been deprecated in {.fn summary_rows}.",
        "*" = "If this was intended for generation of grand summary rows, instead
  use the `grand_summary_rows()` function."
      ))
```

Since this is already broken, it would not be a breaking change to do this. (For consistency, I think it may be preferable?

File: R/summary_rows.R [hunk 3]
```diff
@@ -393,28 +393,19 @@ summary_rows <- function(
...
-      return(data)
-    }
-  }
-
   # Resolve the group names
   groups <-
     resolve_groups(
       expr = {{ groups }},
       vector = available_groups
     )
```
Comment (olivroy) [inline R/summary_rows.R:401]: Moving the resolver before has forced me to add the `isFALSE()` condition in the resolver. The `groups = FALSE` option is undocumented in `summary_rows()` groups argument, but I wonder if I missed some other edge cases?

File: tests/testthat/test-summary_rows.R [hunk 4]
```diff
@@ -635,6 +635,16 @@ test_that("Using `groups = FALSE` in `summary_rows()` returns data unchanged", {
   )
 })
 
+test_that("summary_rows() informs to use grand_summary_rows() if no groups are present (#1292).", {
+  expect_snapshot(
+    gt <- mtcars_short %>%
```
Comment (olivroy) [inline tests/testthat/test-summary_rows.R:640]:
```suggestion
  expect_snapshot(error = TRUE,
    gt <- mtcars_short %>%
```
if we decide to keep this an error.

**Review by olivroy — COMMENTED** [review]
For the moment, I implemented the proposal of #1292, but I think that at this point, it is probably better to deprecate, since this has not worked for a while now.

**Comment by olivroy** [comment]
I will have to review the error message anyway before merging, so I will adjust to an error at the same time if you think it is a good idea.

File: R/resolver.R (hunk 2 above)
Comment (rich-iannone) [reply R/resolver.R:573]: I like it! I'll commit the suggestion.

**Comment by olivroy** [comment]
I am updating locally. Will push a working version shortly!

File: R/summary_rows.R (hunk 3 above)
Comment (rich-iannone) [reply R/summary_rows.R:401]: I think you might be fine here.

**Review by rich-iannone — APPROVED** [review]
LGTM!

**Comment by rich-iannone** [comment]
Thanks! I'll merge once CI is completed.
